2024-11-02 09:44:58,269 ERROR: Error saving recipe: (sqlite3.OperationalError) no such column: recipe.image_url
[SQL: SELECT recipe.id AS recipe_id, recipe.name AS recipe_name, recipe.ingredients AS recipe_ingredients, recipe.instructions AS recipe_instructions, recipe.user_id AS recipe_user_id, recipe.image_url AS recipe_image_url 
FROM recipe 
WHERE recipe.name = ? AND recipe.user_id = ?
 LIMIT ? OFFSET ?]
[parameters: ('Protein-Packed Vegetarian Feast', 1, 1, 0)]
(Background on this error at: https://sqlalche.me/e/20/e3q8) [in /home/kevin-pesenecker/Documents/code/munch/app/routes.py:96]
2024-11-02 09:46:04,631 INFO: Munch startup [in /home/kevin-pesenecker/Documents/code/munch/app/utils/logger.py:15]
2024-11-02 09:46:05,325 INFO: Munch startup [in /home/kevin-pesenecker/Documents/code/munch/app/utils/logger.py:15]
2024-11-02 09:46:06,138 INFO: Munch startup [in /home/kevin-pesenecker/Documents/code/munch/app/utils/logger.py:15]
2024-11-02 09:46:48,217 INFO: Munch startup [in /home/kevin-pesenecker/Documents/code/munch/app/utils/logger.py:15]
2024-11-02 09:46:59,715 INFO: Munch startup [in /home/kevin-pesenecker/Documents/code/munch/app/utils/logger.py:15]
2024-11-02 09:47:12,434 INFO: Munch startup [in /home/kevin-pesenecker/Documents/code/munch/app/utils/logger.py:15]
2024-11-02 09:47:23,081 INFO: Munch startup [in /home/kevin-pesenecker/Documents/code/munch/app/utils/logger.py:15]
2024-11-02 09:47:35,634 INFO: Munch startup [in /home/kevin-pesenecker/Documents/code/munch/app/utils/logger.py:15]
2024-11-02 09:47:41,335 INFO: Munch startup [in /home/kevin-pesenecker/Documents/code/munch/app/utils/logger.py:15]
2024-11-02 09:48:21,737 INFO: Munch startup [in /home/kevin-pesenecker/Documents/code/munch/app/utils/logger.py:15]
2024-11-02 09:48:46,530 INFO: Munch startup [in /home/kevin-pesenecker/Documents/code/munch/app/utils/logger.py:15]
2024-11-02 09:51:10,890 INFO: Munch startup [in /home/kevin-pesenecker/Documents/code/munch/app/utils/logger.py:15]
